### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by andgarzonmal**
Good night, 

This is my project for the desktop version, I used two media queries so that the project can fit on many devices.

I hope everything is ok and thank for your recommendations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,31 +20,51 @@
...
+          </li>
+          <li>
+            <a href="" class="header-btn desktop">Contact</a>
+          </li>
+          <li id=".menu" class="desktop">
+            <a href="#">
+              <i class="material-icons">mail_outline</i>
+            </a>
+          </li>
+          <li id="menu" class="phone">
```
Comment (Omar-Muhamad) [inline index.html:43]: You did amazing job so far there is one required change to mention:

- [ ] Please add the correct margin values for the navbar styling on the left and the right and try to stick as much as possible to the given Figma design.

Figma Design:
image

Your Design:
image

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @andgarzonmal,

Great job styling the desktop version 💪, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (andgarzonmal) [reply index.html]: thanks for your recommendations, I have just fixed the mistake.

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @andgarzonmal,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**